src/scrape_up/icc/icc_rankings.py

```diff
@@ -93,3 +95,83 @@ def player_ranking(self, type, format):
             return response_list
         except:
             return None
+
+    def team_rankings_women(self, format):
+        """
+        Create an instance of `ICC` class.\n
+        Required Params - `format` - "ODI","T20"
+        ```python
+        icc = ICC()
+        icc.team_rankings(format="odi")
+        ```
+        ```js
+        [
+            {
+                "rank":1,
+                "team":"Australia"
+            }
+        ]
+        ```
+        """
+        try:
+            format = format.lower()
+            obj_keys = ["rank", "team"]
+            resposne_list = []
+            url = self.url_women + "team-rankings/" + format
+            response = requests.get(url)
+            soup = BeautifulSoup(response.content, "html.parser")
+            teams = soup.find_all("span", class_="u-hide-phablet")
+            for rank, team in enumerate(teams, 1):
+                obj_values = [rank, team.get_text()]
+                resposne_list.append(dict(zip(obj_keys, obj_values)))
+
+            return resposne_list
+        except:
+            return None
+
+    def player_ranking_women(self, type, format):
+        """
```

Comment:
Test the method and also change the docstring accordingly I think you just copied from the previous methods.

Also, test the endpoints if they are returning data properly or not in the desired format.

Reply:
Data is being returned in proper format and i have updated the docstring